locales/en/base.json

```diff
@@ -1,33 +1,84 @@
 {
+  "menu": {
+    "window": {
```

Comment:
Not 100% sold on the 3 level deep stuff here.. Wondering if we could make it simpler by just having something like `"appmenu"` or something like that? I don't hate it I'm just not sure if we need the extra level, especially seeing as there is another `"window"` group further into the nested items.

Reply:
Fair enough. `app-menu`, `archive-menu`, as top level items are still quite easy to find.